### PR TITLE
Lbsd 1042 - Attach syndicated blog modal

### PR DESCRIPTION
**Pull request by m0g**

### CODE REVIEW

**Comment by mscam** [comment]
:+1:

File: client/app/scripts/liveblog-syndication/directives/attach-syndicated-blogs-modal.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                        return $http({
+                            url: uri,
+                            method: (method == 'DELETE') ? 'DELETE' : 'POST',
+                            data: { consumer_blog_id: consumerBlogId },
+                            headers: {
+                                "Content-Type": "application/json;charset=utf-8"
+                            }
+                        })
+                        .then(function(response) {
+                            console.log('response to delete', response);
```
Comment (djbrd-sourcefabric) [inline client/app/scripts/liveblog-syndication/directives/attach-syndicated-blogs-modal.js]: Can the response here not also be to a post?

Comment (m0g) [inline client/app/scripts/liveblog-syndication/directives/attach-syndicated-blogs-modal.js]: mmm, what?

Comment (m0g) [inline client/app/scripts/liveblog-syndication/directives/attach-syndicated-blogs-modal.js]: How you mean because of the `response to delete`

**Pull request merged**